File: FilmesAPI.FSharp/Data/FilmeContext.fs
namespace FilmesAPI.FSharp.Data

open Microsoft.EntityFrameworkCore
open FilmesAPI.FSharp.Models

type FilmeContext(options: DbContextOptions) =
    inherit DbContext(options)

    [<DefaultValue>]
    val mutable private filmes : DbSet<Filme>
    member this.Filmes
        with get() = this.filmes
        and private set v = this.filmes <- v

File: FilmesAPI.Minimal/Data/FilmeContext.cs
using FilmesAPI.Minimal.Models;
using Microsoft.EntityFrameworkCore;

namespace FilmesAPI.Minimal.Data
{
    public class FilmeContext: DbContext
    {
        public FilmeContext(DbContextOptions<FilmeContext> options) : base(options)
        {

        }

        public DbSet<Filme> Filmes { get; set; }
    }
}
